pkg/cloud/vsphere/services/govmomi/create.go

```diff
@@ -267,23 +267,23 @@ func Create(ctx *context.MachineContext, bootstrapToken string) error {
 }
 
 func findVMByInstanceUUID(ctx *context.MachineContext) (string, error) {
-	ctx.Logger.V(4).Info("finding vm by instance UUID", "instance-uuid", ctx.Machine.UID)
+	ctx.Logger.V(6).Info("finding vm by instance UUID", "instance-uuid", ctx.Machine.UID)
 
 	vmRef, err := ctx.Session.FindByInstanceUUID(ctx, string(ctx.Machine.UID))
 	if err != nil {
 		return "", errors.Wrapf(err, ctx.String())
 	}
 
 	if vmRef != nil {
-		ctx.Logger.V(2).Info("found machine by instance UUID", "vmRef", vmRef.Reference().Value, "instance-uuid", ctx.Machine.UID)
+		ctx.Logger.V(5).Info("found machine by instance UUID", "vmRef", vmRef.Reference().Value, "instance-uuid", ctx.Machine.UID)
```

Comment:
nit, i don't see a real difference between v5 and v6

Reply:
Hi @figo,

It was changed from 2 to 5.

Reply:
i know, but my question is why not 6? as several other logs in this change, or do we really differentiate level 5 and 6?

Reply:
Good question. Honestly I want to see someone write a guide on what to log and at what level. I may create an issue on the topic.